optuna/study.py

```diff
@@ -511,14 +503,60 @@ def create_study(
             raise
 
     study_name = storage.get_study_name_from_id(study_id)
-
-    return Study(
+    study = Study(
         study_name=study_name,
         storage=storage,
         sampler=sampler,
         pruner=pruner,
         direction=direction)
 
+    if direction == 'minimize':
+        _direction = structs.StudyDirection.MINIMIZE
+    elif direction == 'maximize':
+        _direction = structs.StudyDirection.MAXIMIZE
+    else:
+        raise ValueError('Please set either \'minimize\' or \'maximize\' to direction.')
+
+    # TODO(Yanase): Implement maximization.
+    if _direction == structs.StudyDirection.MAXIMIZE:
+        raise ValueError('Optimization direction of study {} is set to `MAXIMIZE`. '
+                         'Currently, Optuna supports `MINIMIZE` only.'.format(study_name))
+
+    study.storage.set_study_direction(study_id, _direction)
+
+    return study
+
+
+def load_study(
```

Comment:
Would you mind adding a unit test like this? https://github.com/pfnet/optuna/blob/master/tests/test_study.py#L470

Reply:
Thank you for your information. I will add a unit test.

Reply:
Added: https://github.com/pfnet/optuna/pull/330/commits/a9a08014aa79d4e05f7b25cf9db040b7cbc3ea69